src/css/base.css

```diff
@@ -136,3 +140,20 @@ button {
 }
 
 /*#endregion Misc */
+
+/* #region Typography */
+.section-heading {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 36px;
+  text-align: center;
+  color: var(--black);
+}
+
+.bold {

```

Comment:
The meaning of `bold` is to provide just `font-weight`. Maybe it should have a different name e.g. `heading-small` or `small-heading`. Also I checked the styles, you missed the `line-height: 1.2; letter-spacing: -0.4px;` for that heading. Please update the class name and add missed properties.

Reply:
The line height and letter spacing is the same as for the normal text that is why I changed the body styling.